mesheryctl/internal/cli/root/system/check.go

```diff
@@ -513,8 +528,101 @@ func (hc *HealthChecker) runComponentsHealthChecks() error {
 }
 
 // runOperatorHealthChecks executes health-checks for Operators
-func (hc *HealthChecker) runOperatorHealthChecks() error {
+func (hc *HealthChecker) runOperatorHealthChecks(k8scontext string) error {
```

Comment:
is this parameter even required?

Reply:
Thanks will do the changes
I'm still refactoring the code